src/quotes.js

```diff
@@ -0,0 +1,115 @@
+const quotes = [
+  {
+    quote: 'God forever geometrizes.',
+    by: 'Plato',
+  },
+  {
+    quote: 'Wherever there is number, there is beauty.',
+    by: 'Proclus',
+  },
+  {
+    quote: 'The book of nature is written in the language of mathematics.',
+    by: 'Galileo',
+  },
+  {
+    quote: 'Mathematics is the Queen of the Sciences.',
+    by: 'Carl Friedrich Gauss',
+  },
+  {
+    quote:
+        'But of all the sciences cultivated by mankind, none are more useful than mathematics, to call forth a spirit of enterprise and enquiry.',
+    by: 'Consider Sterry',
+  },
```

Comment:
- [ ] Please why are you creating all these quotes which are not required in this project  :relaxed: